### PR TITLE
WIP: Stage 3 Remaster

### PR DESCRIPTION
**Pull request by Akaricchi**
This includes, and will be a continuation of, #197.

### CODE REVIEW

File: src/stages/stage3/nonspells/midboss_nonspell_1.c [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+					.pos = p->pos,
+					.color = RGBA(0.8, 0.6, 0.6, 0),
+					.timeout = 100,
+					.draw_rule = pdraw_timeout_scalefade(0, 0.8, 1, 0),
+					.move = move_asymptotic_simple(p->move.velocity + rng_dir(), 0.1)
+				);
+
+				real offset = global.frames/15.0;
+				if(global.diff > D_Hard && global.boss) {
+				offset = M_PI + carg(global.plr.pos - global.boss->pos);
```
Comment (Akaricchi) [inline src/stages/stage3/nonspells/midboss_nonspell_1.c:74]: misleading indentation

**Comment by StarWitch** [comment]
This stage runs, but there's a lot of dead-time since we decided to redo a lot of the stage mob patterns. That'll need to get done before merge.

**Comment by laochailan** [comment]
yes. I kind of burned myself out on it when I tried to redo patterns. If you want you can have a shot at it.

**Comment by StarWitch** [comment]
No worries at all, just commenting so we have an idea of where each PR is at.

**Comment by Akaricchi** [comment]
Merged as 558541e2cc300df2e07a8217547b46db34442541

**Pull request closed without merging**